generator.yml

```diff
@@ -175,7 +175,7 @@ builds:
   - {<< : *arm32_v7,      << : *stable-5_10,      << : *llvm,            boot: true,  llvm_version: *llvm_tot}
   - {<< : *arm32_v7_t,    << : *stable-5_10,      << : *llvm,            boot: true,  llvm_version: *llvm_tot}
   - {<< : *arm32_allmod,  << : *stable-5_10,      << : *llvm,            boot: false, llvm_version: *llvm_tot}
-  - {<< : *arm32_allno,   << : *stable-5_10,      << : *llvm,            boot: false, llvm_version: *llvm_tot}
+  - {<< : *arm32_allno,   << : *stable-5_10,      << : *llvm_full,       boot: false, llvm_version: *llvm_tot}
```

Comment:
Can other things on 5.10 be enabled or is this the only one for now?

Reply:
Not yet, so far for 5.10 we need at least:
3c9f5708b7aed6a963e2aefccbd1854802de163e v5.11-rc1
0b1674638a5c69cbace63278625c199100955490 v5.11-rc1
67e3f828bd4bf5e4eb4214dc4eb227d8f1c8a877 v5.11-rc1